packages/webapp/src/components/ActionMenu/styles.module.scss

```diff
@@ -31,7 +31,7 @@
     left: 50%;
 
     &.iconCount_4 {
-      width: 408px;
+      width: 424px;
```

Comment:
@SayakaOno this was my first thought for fitting the longer label, but now actually I think I slightly prefer the look of decreasing the gap from 8px to 4px; curious which you think is better?

Reply:
I think I set the gap to 8px because the space looked too narrow with 4px.
![Screenshot 2024-03-20 at 3 20 52 PM](https://github.com/LiteFarmOrg/LiteFarm/assets/33141219/3b873670-85ac-4a2b-865a-4c4a15f79b49)

To me, an 8 pixel gap still seems better when thinking about other languages...